tests/testthat/_snaps/epiparam.md

```diff
@@ -8,15 +8,15 @@
       Number of diseases: 23
       Number of delay distributions: 95
       Number of offspring distributions: 10
-      Number of studies in library: 57
+      Number of studies in library: 45
       <Head of library>
             disease         epi_distribution prob_distribution
       1  Adenovirus        incubation_period             lnorm
       2 Chikungunya        incubation_period             lnorm
       3    COVID-19 hospitalisation_to_death           weibull
       4    COVID-19 hospitalisation_to_death             lnorm
       5    COVID-19        incubation_period              <NA>
-      6    COVID-19        incubation_period           weibull
```

Comment:
This entry has changed from Weibull to `NA` but checking locally shows it's an issue of row reordering.

Reply:
Yes, I've also checked this and the data is the same it is just the ordering that changed. I think this might be due to the author information changing: https://github.com/epiverse-trace/epiparameter/blob/main/R/epiparam.R#L53-L58